ecosystem/sep-0040.md

```diff
@@ -0,0 +1,160 @@
+## Preamble
+
+```
+SEP: To Be Assigned
+Title: Non-Fungible Token Standard
+Author: George Kudrayvtsev <george@stellar.org>
+Track: Standard
+Status: Draft
+Created: 2021-01-31
+Updated: 2021-01-31
+Version: 0.1.0
+Discussion: TBD
+```
+
+## Simple Summary
+This SEP defines a standard interface to describe non-fungible tokens (NFTs) on the Stellar network, including a distribution mechanism and metadata fields. It draws inspiration from the [EIP-721 standard](https://eips.ethereum.org/EIPS/eip-721).
```

Comment:
Suggest using the same terms here as you do later on, so that they are clearly relatable as the same concepts.
```suggestion
This SEP defines a standard interface for non-fungible tokens (NFTs) on the Stellar network, including a issuance model and metadata fields. It draws inspiration from the [EIP-721 standard](https://eips.ethereum.org/EIPS/eip-721).
```